src/app/containers/MediaPlayer/helpers/metadata/index.js

```diff
@@ -4,7 +4,15 @@ import pathOr from 'ramda/src/pathOr';
 const DEFAULT_IMAGE_RES = '1024x576';
 
 export const getThumbnailUri = aresMetadataBlock => {
-  const imageUrl = pathOr('', ['model', 'imageUrl'], aresMetadataBlock);
+  let imageUrl = pathOr('', ['model', 'imageUrl'], aresMetadataBlock);
+
+  if (imageUrl.includes('http')) {
```

Comment:
What if the imageUrl includes `https` - will this still work?

Reply:
It will - but this is a good test scenario

Reply:
I've added a test for this - 'should return a valid thumbnail Uri for https assets'. See if you think it's robust enough?